packages/flutter/lib/src/widgets/debug.dart

```diff
@@ -48,6 +48,9 @@ bool debugPrintScheduleBuildForStacks = false;
 /// This can help track down framework bugs relating to the [GlobalKey] logic.
 bool debugPrintGlobalKeyedWidgetLifecycle = false;
 
+/// Adds [Timeline] events for every Widget built.
```

Comment:
If you could add some docs here on how to actually use that information (e.g. pointing to the observatory timeline UI, or pointing to a doc that talks about how to use timeline events), that would be great.